app/src/main/java/com/example/garden_planner/PlantAdditionActivity.java

```diff
@@ -1,17 +1,42 @@
 package com.example.garden_planner;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
 
+import com.example.garden_planner.adapters.PlantAdditionAdapter;
+import com.example.garden_planner.adapters.PlantInBedAdapter;
 import com.example.garden_planner.databinding.ActivityMainBinding;
 import com.example.garden_planner.databinding.ActivityPlantAdditionBinding;
+import com.example.garden_planner.models.Garden;
+import com.example.garden_planner.models.Plant;
+import com.example.garden_planner.models.PlantInBed;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.SaveCallback;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class PlantAdditionActivity extends AppCompatActivity {
 
     private ActivityPlantAdditionBinding binding;
 
+    private Garden garden;
+    private Button btCreatePlant;
```

Comment:
nit: let's write out createPlantButton, plantNameEditText, plantRecyclerView, etc instead so it's easier to follow too

Reply:
for new variables, I'll implement this naming convention, but can I complete the renaming of already-made variables later? I want to keep making progress and fix the nits when it's a little slower